docs/runtime_CLI.md

```diff
@@ -320,6 +320,50 @@ burst_size is the number of packets.
 ```
 TODO: port_add
 TODO: port_remove
+```
+
+### pvs_add, pvs_clear, pvs_get, pvs_remove
+
+These commands are used to control an instance of a P4 parser value
+set. `pvs_add` is used to add an entry to a value set, `pvs_remove` is used to
+remove an entry, `pvs_clear` is used to remove *all* entries, and `pvs_get` is
+used to list all entries.
+
+P4 allows a value_set to take a struct as the type parameter, such as in the
+following example:
+```
+struct vsk_t {
+    bit<16> f1;
+    bit<7> f2;
+}
+value_set<vsk_t>(4) pvs;
+select (<X>) {
+    pvs: accept;
+    _: reject;
+}
+```
+When adding or removing an entry, you must provide a single integer value, which
+must fit within the value set's "compressed bitwidth", or the CLI will display
+an error. When a value set's entries consitute of multiple individual fields, as
```

Comment:
maybe "consist of" instead of "consitute of" ?  "consitute" is not a word, in any case.

Reply:
Fixed, thanks for pointing this out